src/test/kotlin/fp/Lambda.kt

```diff
@@ -0,0 +1,25 @@
+package fp
+
+class Lambda {
+    val numbers: List<Int> = listOf(1, 2, 3, 4, 5, 6)
+
+    fun sumAll(numbers: List<Int>): Int = sumIf(numbers) { true }
+
+    fun sumAllEven(numbers: List<Int>): Int = sumIf(numbers) {
+        it % 2 == 0
+    }
```

Comment:
아래 sumIf를 아예 제거하고, kotlin collection에서 제공하는 메소드들을 활용해보아도 괜찮을 것 같네요!

```suggestion
    this.numbers.filter { it % 2 == 0 }.sum()
```